### PR TITLE
Pull request for Rvfbrnet 36 fix

### PR DESCRIPTION
**Pull request by AndriiKatsubo**

### CODE REVIEW

File: MedicalDocManagment.WebUI/index.html [hunk 1]
```diff
@@ -19,17 +19,6 @@
     <script src="./dist/js/angular2/system.src.js"></script>
     <script src="dist/lib/npmlibs/ng2-pagination/dist/ng2-pagination-bundle.js"></script>
     <script src="systemjs.config.js"></script>
-    
-    <!--<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">    
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
-    
-    <base href="/">
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/index.html]: Please revert this change.
We need this code for testing

File: MedicalDocManagment.WebUI/app/users-list/page/item.component.ts [hunk 2]
```diff
@@ -7,8 +7,7 @@ import UserModel from '../../models/usermodel'
 @Component({
   moduleId: module.id,
   selector: '[item]',
-  templateUrl: 'views/item.component.html',
-  styleUrls: ['views/item.component.css'],
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/app/users-list/page/item.component.ts]: Why did you delete css files?

**Review by artempanko — APPROVED** [review]

File: MedicalDocManagment.WebUI/app/users-list/page/item.component.ts (hunk 2 above)
Comment (AndriiKatsubo) [reply MedicalDocManagment.WebUI/app/users-list/page/item.component.ts]: That's an outdated code — I've returned css files in last commits.

Comment (AndriiKatsubo) [reply MedicalDocManagment.WebUI/app/users-list/page/item.component.ts]: https://github.com/Social-projects-Rivne/medical_doc_managment/pull/27/files
